packages/animations/lib/src/open_container.dart

```diff
@@ -362,12 +386,40 @@ class _OpenContainerRoute extends ModalRoute<void> {
 
   @override
   final Duration transitionDuration;
+  final ContainerTransitionType transitionType;
 
   final Tween<double> _elevationTween;
   final ShapeBorderTween _shapeTween;
   final _FlippableTweenSequence<Color> _colorTween;
 
-  final _FlippableTweenSequence<double> _closedOpacityTween =
+  // Fade transition opacity tweens
+  static final _FlippableTweenSequence<double> _fadeOpenOpacityTween =
+      _FlippableTweenSequence<double>(<TweenSequenceItem<double>>[
+    TweenSequenceItem<double>(
+      tween: ConstantTween<double>(0.0),
+      weight: 1 / 5,
+    ),
+    TweenSequenceItem<double>(
+      tween: Tween<double>(begin: 0.0, end: 1.0),
+      weight: 1 / 5,
+    ),
+    TweenSequenceItem<double>(
+      tween: ConstantTween<double>(1.0),
+      weight: 3 / 5,
+    ),
+  ]);
+  static final _FlippableTweenSequence<double> _fadeClosedOpacityTween =
+      _FlippableTweenSequence<double>(
+    <TweenSequenceItem<double>>[
+      TweenSequenceItem<double>(
+        tween: ConstantTween<double>(1.0),
+        weight: 1,
+      ),
+    ],
+  );
+
+  // Fade through transition opacity tweens
```

Comment:
Can you update all the weights of the existing tweens?

Reply:
Done